assembl/static2/js/app/components/debate/common/postsFilter/filtersCategoryLabel.jsx

```diff
@@ -0,0 +1,28 @@
+// @flow
+import * as React from 'react';
+import { Translate } from 'react-redux-i18n';
+
+type Props = {
+  labelMsgId: string
+};
+
+class FiltersCategoryLabel extends React.Component<Props> {
+  render() {
+    const categoryLabelStyle = {
```

Comment:
Hmmm...adhoc css in js can be a bit dangerous if during refactoring of the styling in the css misses the rules here. I'm not sure how to feel about this...

Reply:
Okay no, please move this css into the css folder and use classnames. Adhoc changes in CSS will cause a ton of headaches downstream, especially with the ambition of cleanup of CSS once the frontend V2 is removed from V2.

For sake of reduced comments, please move all css-in-js to the css folder, to the appropriate place and use classnames.

Reply:
What is the appropriate place ? 
the css is split up between files, sometimes a same class is in multiple places, and there is no indications of good practice on where this should go. I'd rather not create new classes and keep styles locales than add some classes on random files

Reply:
"Adhoc changes in CSS will cause a ton of headaches downstream" < ?

Reply:
I also found hundred of occurrences of style used directly on a component, often with a className, so I think we're pretty safe if i use some local style without classname on a component ?

Reply:
We didn't have pattern of putting entire css of a component inline in the `style`. We did use `style` for dynamic css such as colour changes, position changes, etc. You should aim to create an scss class following the (not so great) pattern in the css folder: create a new file for your components such as `filter.scss` and add your css as a class there. Then use the classnames. Webpack will load your css for you. You just have to ensure you `@import` your new file in `_assembl_base_styles.scss` for webpack to load it up

Reply:
Ok, tried it, I can't, there are too many embedded classes at different places that conflict with my css if I put it in a separate file, so I will close this PR for now and I will come back to it when the css refacto is done